openssl-dynamic/src/main/c/ssl.c

```diff
@@ -1553,16 +1553,21 @@ TCN_IMPLEMENT_CALL(jlong, SSL, makeNetworkBIO0)(TCN_STDARGS,
         return 0;
     }
 
-    if (max_bio_size < 0) {
-        tcn_ThrowException(e, "max_bio_size < 0");
+    if (maxInternalBIOSize < 0) {
```

Comment:
@Scottmitch let us just keep it as it is as this one will be called only very infrequently anyway.